src/aks-preview/linter_exclusions.yml

```
@@ -63,6 +63,11 @@ aks create:
     enable_ai_toolchain_operator:
       rule_exclusions:
         - option_length_too_long
+    enable_secure_boot:
+      rule_exclusions:
+        - option_length_too_long
+    enable_vtpm:
+        - option_length_too_long
```

```suggestion
    enable_vtpm:
      rule_exclusions:
        - option_length_too_long
```
CI fails https://dev.azure.com/azclitools/public/_build/results?buildId=127901&view=logs&j=e2e497a5-ba27-5eb9-f39e-d929d56974a6&t=61cc2d02-f6f4-5007-71e9-5651933c3fe7&l=149

---

fixed